hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -9,7 +9,7 @@ This page explains how to boot your Raspberry Pi from a USB mass storage device
 The Raspberry Pi Pi 400 and newer Raspberry Pi 4B boards support USB boot by default. On earlier Raspberry Pi 4B boards, or to select alternate boot modes, the bootloader must be updated.
 
 ### Using Raspberry Pi Imager to update the bootloader (recommended)
```

Comment:
Instead of duplicating the instructions here, I wonder if it's better to just link to the RPi Imager instructions in `booteeprom.md` ? :shrug:

Reply:
I think that would probably be ok e.g. link to an anchor for the imager in booteeprom.md with a sentence to say select "USB boot"

Reply:
done